packages/cli/src/fwoosh.ts

```diff
@@ -445,8 +450,13 @@ export class Fwoosh implements FwooshClass {
         },
       ],
       optimizeDeps: {
-        entries: [require.resolve("@fwoosh/app/index.html")],
-        exclude: ["@fwoosh/*", "@fwoosh/components"],
+        entries: [
+          require.resolve("@fwoosh/app/index.html"),
+          ...stories.map((s) => path.resolve(s)),
+          ...panels.map((p) => p.filepath),
+          ...toolbarControls.map((p) => p.filepath),
```

Comment:
Add all stories and plugins as entries. Makes the app not reload as things are lazy loaded